app/src/main/cpp/skyline/applet/error_applet.h

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+// Copyright © 2020 Ryujinx Team and Contributors (https://github.com/ryujinx/)
+
+#pragma once
+
+#include <services/am/applet/IApplet.h>
+#include "common/language.h"
+
+namespace skyline::applet {
+    /**
+     * @brief The ErrorApplet class is used to show errors to the user
```

Comment:
`The Error Applet is utilized by the guest to display an error to the user, this class reimplements that and prints the error to the log`